boltons/formatutils.py

```diff
@@ -41,6 +41,7 @@
 import re
 from string import Formatter
 
+from .namedutils import namedtuple
```

Comment:
can you change this to:

```python
try:
    from boltons.namedutils import namedtuple
except ImportError:
    from collections import namedtuple
```

This will allow the module to continue to be used both inside and outside of the boltons package.